src/tax-lot/tax-lot.service.ts

```diff
@@ -78,77 +39,15 @@ export class TaxLotService {
 
   async findTaxLotByBblGeoJson(bbl: string) {
     if (this.featureFlagConfig.useDrizzle) {
-      let result: SelectTaxLotSpatial | undefined;
-      try {
-        result = await this.db.query.taxLot.findFirst({
-          columns: {
-            bbl: true,
-            block: true,
-            lot: true,
-            address: true,
-          },
-          extras: {
-            geometry: sql<string>`ST_AsGeoJSON(${taxLot.wgs84}, 6)`.as(
-              "geometry",
-            ),
-          },
-          where: (taxLot, { eq }) => eq(taxLot.bbl, bbl),
-          with: {
-            borough: true,
-            landUse: true,
-          },
-        });
-      } catch {
-        throw DataRetrievalException;
-      }
-      if (result === undefined) throw ResourceNotFoundException;
-      const geometry = JSON.parse(result.geometry) as MultiPolygon;
-      return {
-        type: "Feature",
-        id: result.bbl,
-        properties: {
-          bbl: result.bbl,
-          borough: result.borough,
-          block: result.block,
-          lot: result.lot,
-          address: result.address,
-          landUse: result.landUse,
-        },
-        geometry,
-      };
+      return await this.taxLotRepo.taxLotByBblGeoJson(bbl);
     } else {
       return this.taxLotRepository.findOne({ bbl });
     }
   }
 
   async findZoningDistrictByTaxLotBbl(bbl: string) {
```

Comment:
@TangoYankee  The function name is also singular, shall I pluralize this as well?

Reply:
We'll refactor this in a follow-up ticket.
1) It's pre-existing code and is technically out of scope of the acceptance criteria
2) We'll likely want to make several changes to this function (and several others) but we're not quite sure yet what they'll be. Better to get it in one go in the follow-up ticket.